code/renderers/vue3/src/docs/extractArgTypes.ts

```diff
@@ -283,17 +283,11 @@ export const convertVueComponentMetaProp = (
       };
     }
 
-    // recursively/deeply convert all properties of the object
     case 'object':
       return {
         name: 'object',
-        value: Object.entries(schema.schema ?? {}).reduce<Record<string, SBType>>(
-          (obj, [propName, propSchema]) => {
-            obj[propName] = convertVueComponentMetaProp(propSchema);
-            return obj;
-          },
-          {}
-        ),
+        // Storybook does not generate controls for object properties so we don't need to recursively map the object schema here
```

Comment:
I'm not entirely sure what you mean here. While it's true that complex objects won't have proper controls, Storybook does generate controls for simple objects (JSON object controls). However, it doesn't require or use nested schemas. I think you meant controls for the properties of objects rather than props of the object type.

Reply:
Yes exactly, I updated the comment so I hope its clearer now :)